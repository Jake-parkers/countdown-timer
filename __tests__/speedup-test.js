import React from "react";
import Speedup from "../components/speedup";

import renderer from "react-test-renderer";

describe("<Speedup/> test", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Speedup
          minuteCounter={9}
          secondCounter={59}
          speedUp={1.9}
          isCountingDown={false}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
