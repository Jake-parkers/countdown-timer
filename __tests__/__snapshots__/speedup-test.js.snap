// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Speedup/> test renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
      "padding": 20,
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "borderColor": "#fff",
        "borderRadius": 4,
        "borderWidth": 2,
        "height": 30,
        "margin": 10,
        "opacity": 1,
        "padding": 5,
        "width": 40,
      }
    }
  >
    <View
      style={Object {}}
    >
      <Text
        style={
          Object {
            "color": "#fff",
            "textAlign": "center",
          }
        }
      >
        1.0x
      </Text>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "borderColor": "#fff",
        "borderRadius": 4,
        "borderWidth": 2,
        "height": 30,
        "margin": 10,
        "opacity": 1,
        "padding": 5,
        "width": 40,
      }
    }
  >
    <View
      style={Object {}}
    >
      <Text
        style={
          Object {
            "color": "#fff",
            "textAlign": "center",
          }
        }
      >
        1.5x
      </Text>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "borderColor": "#fff",
        "borderRadius": 4,
        "borderWidth": 2,
        "height": 30,
        "margin": 10,
        "opacity": 1,
        "padding": 5,
        "width": 40,
      }
    }
  >
    <View
      style={Object {}}
    >
      <Text
        style={
          Object {
            "color": "#fff",
            "textAlign": "center",
          }
        }
      >
        2x
      </Text>
    </View>
  </View>
</View>
`;
